course6/toyapp_thomasdhuys/drawlib/render_object.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+#include <string>
+
+#include "primitives.h"
+
+namespace Draw {
+
+using namespace Draw::Primitives;
+
+class RenderObject {
```

Comment:
* The name of the class is very generic and doesn't really reveal it's purpose. Maybe "Scene" or "World" would be better?
* The class doesn't add much value. You can just provide a function that iterates over the container and calls ToString on every element.

Reply:
- Scene sounds like a better name indeed.
- Meant to use this object as an interface between the "Drawer" which generates the scene and the "Renderer" which puts the scene on the screen